tensorflow/lite/special_rules.bzl

```diff
@@ -72,9 +72,7 @@ def tflite_extra_gles_deps():
 
 def tflite_ios_lab_runner(version):
     """This is a no-op outside of Google."""
-
```

Comment:
We don't need this and https://github.com/bazelbuild/rules_apple/pull/757 if we use an archive version of `xctestrunner`
```
    tf_http_archive(
        name = "xctestrunner",
        strip_prefix = "xctestrunner-0.2.15",
        sha256 = "b789cf18037c8c28d17365f14925f83b93b1f7dabcabb80333ae4331cf0bcb2f",
        urls = tf_mirror_urls("https://github.com/google/xctestrunner/archive/refs/tags/0.2.15.tar.gz"),
    )
```

Reply:
This test runner isn't enough on its own, you still need the rules in rules_apple, but either way I think you need this I assume because it's being changed internally?

Reply:
Yeah, I'm also upgrading rules_apple. At least all macOS presubmits are passing.